src/Modal/index.jsx

```diff
@@ -9,6 +10,11 @@ import Icon from '../Icon';
 import newId from '../utils/newId';
 import Variant from '../utils/constants';
 
+
+const modalRootId = 'modal-root';
+let modalRoot = document.getElementById(modalRootId);
```

Comment:
Having this outside the scope of Modal makes me nervous - what happens if somebody deletes this element in the DOM? `modalRoot` will still probably be defined right?